internal/cmd/controller/gitops/reconciler/gitjob_test.go

```diff
@@ -214,12 +260,19 @@ func TestReconcile_LatestCommitIsOkay(t *testing.T) {
 		},
 	).Times(1)
 
+	recorderMock := mocks.NewMockEventRecorder(mockCtrl)
+	recorderMock.EXPECT().Event(&gitRepoMatcher{gitRepo},
+		fleetevent.Normal,
+		"GotNewCommit",
+		"1883fd54bc5dfd225acf02aecbb6cb8020458e33")
```

Comment:
nit: this kind of formatting is more idiomatic (same remark around L404 and L471):
```suggestion
	recorderMock.EXPECT().Event(
	        &gitRepoMatcher{gitRepo},
		fleetevent.Normal,
		"GotNewCommit",
		"1883fd54bc5dfd225acf02aecbb6cb8020458e33",
		)
```

Reply:
changed